### PR TITLE
feat: Create privacy.html

### PR DESCRIPTION
**Pull request by teolemon**
### What
- feat: Create privacy.html

### CODE REVIEW

**Review by CharlesNepote — APPROVED** [review]

File: lang/en/texts/privacy.html [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+<p>In summary:&nbsp;Open Food Facts is a collaborative project whose aim is to promote food transparency by providing an open product database. To make this database available and allow the community to contribute to it, Open Food Facts processes and stores for a limited period, your personal data such as user name, user identifier, passwords, email address and IP for which you have the right of access and information, right of opposition, right of rectification of data, and the right to limitation of processing. If you plan to exercise one of these rights, you can contact us by email: <a href="mailto:privacy@openfoodfacts.org">privacy@openfoodfacts.org</a></p>
+</td>
+</tr>
+</tbody>
+</table>
+<ol start="1">
+<li>
+<h1>Introduction</h1>
+</li>
+</ol>
```
Comment (alexgarel) [inline lang/en/texts/privacy.html:16]: It's not allowed to put an h1 inside a ol/li.

Moreover it does not output as you would expect:

image

File: lang/en/texts/privacy.html [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+<p>For the security of your personal data, Open Food Facts has implemented good security practices:</p>
+<ul>
+<li>Password hashing;</li>
+<li>Monitoring good practices in development;</li>
+<li>Authentication by SSH key, access to servers limited to a small trusted group, limitation of the attack surface, use of firewalls, etc.</li>
+</ul>
+<ol start="7">
+<li>How to exercise your rights?&nbsp;</li>
```
Comment (alexgarel) [inline lang/en/texts/privacy.html:359]: Should be a h1

File: lang/en/texts/privacy.html [hunk 3]
```diff
@@ -0,0 +1,417 @@
...
+</li>
+</ol>
+<p>To ensure a better protection, Open Food Facts reserves the right to modify and/or adapt this confidentiality policy following regulations and internal best practices. For each substantial modification, Open Food Facts will inform you at least thirty days before their application.</p>
+<p>As a collaborative project, Open Food Facts remains available to receive your suggestions regarding this privacy policy, which you can send by email to <a href="mailto:privacy@openfoodfacts.org">privacy@openfoodfacts.org</a></p>
+<ol start="10">
+<li>
+<h1>What are your rights ?</h1>
+</li>
+</ol>
+<p>The right of access and to information</p>
```
Comment (alexgarel) [inline lang/en/texts/privacy.html:382]: This and other below have to be h2

File: lang/en/texts/privacy.html (hunk 1 above)
Comment (alexgarel) [reply lang/en/texts/privacy.html:16]: you even have the summary which is displayed after Introduction, I don't know why !

**Review by alexgarel — CHANGES_REQUESTED** [review]
There are problems with the layout.

File: lang/en/texts/privacy.html (hunk 2 above)
Comment (teolemon) [inline lang/en/texts/privacy.html:359]:
```suggestion
<h1>How to exercise your rights?&nbsp;</h1>
```

File: lang/en/texts/privacy.html (hunk 3 above)
Comment (teolemon) [inline lang/en/texts/privacy.html:382]:
```suggestion
<h2>The right of access and to information</h2>
```

File: lang/en/texts/privacy.html [hunk 4]
```diff
@@ -0,0 +1,417 @@
...
+<p>You have the right to ask whether Open Food Facts processes your personal data and to request the communication of a copy if necessary. You can also request the portability of some of this data in a readable format.</p>
+<p>You have the right to be informed about the purposes for which your data is collected, the way in which they are processed, used and stored.</p>
+<p>The right to object</p>
```
Comment (teolemon) [inline lang/en/texts/privacy.html:385]:
```suggestion
<h2>The right to object</h2>
```

File: lang/en/texts/privacy.html [hunk 5]
```diff
@@ -0,0 +1,417 @@
...
+<p>You can object to your data being used for a specific objective provided that this data is not mandatory to achieve this objective.</p>
+<p>The right to rectification of data</p>
```
Comment (teolemon) [inline lang/en/texts/privacy.html:387]:
```suggestion
<h2>The right to rectification of data</h2>
```

File: lang/en/texts/privacy.html [hunk 6]
```diff
@@ -0,0 +1,417 @@
...
+<p>You have the right to request the rectification of your inaccurate or incomplete information in order to avoid the processing of any incorrect data or to complete missing data.</p>
+<p>The right to erasure of data</p>
```
Comment (teolemon) [inline lang/en/texts/privacy.html:389]:
```suggestion
<h2>The right to erasure of data</h2>
```

File: lang/en/texts/privacy.html [hunk 7]
```diff
@@ -0,0 +1,417 @@
...
+<p>You have the right to request the deletion of one or more of your personal data, specifying which data it concerns.</p>
+<p>For personal data that is not technically erasable, Open Food Facts will use the scrambling technique when it is the subject of an erasure request.</p>
+<p>The right to erasure is waived if the retention of the data is necessary for the establishment, exercise or defense of legal rights.</p>
+<p>The right to restriction of data processing</p>
```
Comment (teolemon) [inline lang/en/texts/privacy.html:393]:
```suggestion
<h2>The right to restriction of data processing</h2>
```

File: lang/en/texts/privacy.html [hunk 8]
```diff
@@ -0,0 +1,417 @@
...
+<ol start="8">
+<li>
+<h1>Dispute resolution</h1>
+</li>
+</ol>
+<p>In the event of a dispute between you and Open Food Facts regarding the processing of your personal data, please contact us by email at the following address <a href="mailto:privacy@openfoodfacts.org">privacy@openfoodfacts.org</a>. We will respond as soon as possible.</p>
+<p>However, in the event of an unsatisfactory response or no response within a maximum of three months, you have the right to contact the CNIL via the following link <a href="https://www.google.com/url?q=https://www.cnil.fr/fr/saisir-la-cnil&amp;sa=D&amp;source=editors&amp;ust=1707329438937473&amp;usg=AOvVaw32ebNM1Cuw7Pn7wIoJiFzi">https://www.cnil.fr/fr/saisir-la-cnil</a>&nbsp;or <a href="https://www.google.com/url?q=https://www.cnil.fr/fr/plaintes&amp;sa=D&amp;source=editors&amp;ust=1707329438937876&amp;usg=AOvVaw2477y8DFLVjQS0a0HWw-or">https://www.cnil.fr/fr/plaintes</a>&nbsp;</p>
```
Comment (teolemon) [inline lang/en/texts/privacy.html:369]:
```suggestion
<p>However, in the event of an unsatisfactory response or no response within a maximum of three months, you have the right to contact the CNIL via the following link <a href="https://www.cnil.fr/fr/saisir-la-cnil">https://www.cnil.fr/fr/saisir-la-cnil</a>&nbsp;or <a href="https://www.cnil.fr/fr/plaintes">https://www.cnil.fr/fr/plaintes</a>&nbsp;</p>
```

File: lang/en/texts/privacy.html [hunk 9]
```diff
@@ -0,0 +1,417 @@
...
+<tr>
+<td colspan="1" rowspan="1">
+<p>In summary:&nbsp;Open Food Facts is a collaborative project whose aim is to promote food transparency by providing an open product database. To make this database available and allow the community to contribute to it, Open Food Facts processes and stores for a limited period, your personal data such as user name, user identifier, passwords, email address and IP for which you have the right of access and information, right of opposition, right of rectification of data, and the right to limitation of processing. If you plan to exercise one of these rights, you can contact us by email: <a href="mailto:privacy@openfoodfacts.org">privacy@openfoodfacts.org</a></p>
+</td>
+</tr>
+</tbody>
+</table>
+<ol start="1">
+<li>
+<h1>Introduction</h1>
```
Comment (teolemon) [inline lang/en/texts/privacy.html:14]:
```suggestion
<h1>1. Introduction</h1>
```
@alexgarel let's do it the non-HTML way

Comment (alexgarel) [reply lang/en/texts/privacy.html:14]: yes

File: lang/en/texts/privacy.html [hunk 10]
```diff
@@ -0,0 +1,417 @@
...
+<p>Email : <a href="mailto:contact@openfoodfacts.org">contact@openfoodfacts.org</a></p>
+<p>Our philosophy</p>
+<p>Open Food Facts assures its visitors and contributors that it does not profile people based on their age, behaviors or habits without their explicit consent. Likewise, it does not communicate your personal data to third parties for commercial purposes.</p>
+<p>Thus, Open Food Facts only collects the personal information strictly necessary to achieve its objectives and under no circumstances uses this information for purposes other than those for which it is collected.</p>
+<p>Concerned about the confidentiality of your data, Open Food Facts not only strive to comply with the legislation in force in Europe<sup><a href="#cmnt2">[b]</a></sup>, it also takes into account other international regulations.</p>
+<ol start="2">
+<li>
+<h1>Your personal data and their use by Open Food Facts</h1>
+</li>
+</ol>
```
Comment (teolemon) [inline lang/en/texts/privacy.html:29]:
```suggestion
<h1>2. Your personal data and their use by Open Food Facts</h1>
```

**Review by alexgarel — APPROVED** [review]
I approve to unblock you when fix are done.

File: lang/en/texts/privacy.html [hunk 11]
```diff
@@ -0,0 +1,417 @@
...
+<p>Legitimate interest:establishment, exercise or defense of a legal right</p>
+</td>
+</tr>
+</tbody>
+</table>
+<ol start="3">
+<li>
+<h1>Personal data processed by OpenFoodFacts</h1>
+</li>
+</ol>
```
Comment (teolemon) [inline lang/en/texts/privacy.html:318]:
```suggestion
<h1>3. Personal data processed by OpenFoodFacts</h1>
```

File: lang/en/texts/privacy.html [hunk 12]
```diff
@@ -0,0 +1,417 @@
...
+<li>Nutritional quality (amount of salt, sugar, fat, fatty acid);</li>
+<li>Food processing;</li>
+<li>Allergens and ingredients contained in foods;</li>
+</ul>
+<p>This data is stored in your browser and mobile applications (secure storage). They are neither communicated nor transmitted to third parties.&nbsp;</p>
+<ol start="4">
+<li>
+<h1>Public and private information:</h1>
+</li>
+</ol>
```
Comment (teolemon) [inline lang/en/texts/privacy.html:335]:
```suggestion
<h1>4. Public and private information:</h1>
```

File: lang/en/texts/privacy.html [hunk 13]
```diff
@@ -0,0 +1,417 @@
...
+<p>The&nbsp;username and user ID can be visible to other users of our services unlike email which is confidential. The country in which the contributor added and/or modified a product is also visible and can be cross-referenced with their username.</p>
+<p>The database is published in its entirety under an open data license, which is one of the primary goals of the project. It only contains the user's identifier as well as the products he added and modified, products possibly located in a country.</p>
+<p>When deleting the user account, the user ID is replaced with a random but unique username for that user. All other personal data is deleted.</p>
+<ol start="5">
+<li>
+<h1>Cookies and other terminal identifiers</h1>
+</li>
+</ol>
```
Comment (teolemon) [inline lang/en/texts/privacy.html:343]:
```suggestion
<h1>5. Cookies and other terminal identifiers</h1>
```

File: lang/en/texts/privacy.html [hunk 14]
```diff
@@ -0,0 +1,417 @@
...
+<p>Cookies are small files that a site or its service providers transfer to your computer's hard drive through your browser.</p>
+<p>These cookies are used to authenticate you, understand and save your preferences for future visits to our sites and compile aggregate data for traffic and interaction on our websites. These cookies are among those that do not require your consent.<sup><a href="#cmnt5">[e]</a></sup><sup><a href="#cmnt6">[f]</a></sup></p>
+<p>Open Food Facts does not transfer these cookies to third parties.</p>
+<ol start="6">
+<li>
+<h1>Security measures</h1>
+</li>
+</ol>
```
Comment (teolemon) [inline lang/en/texts/privacy.html:351]:
```suggestion
<h1>6. Security measures</h1>
```

File: lang/en/texts/privacy.html [hunk 15]
```diff
@@ -0,0 +1,417 @@
...
+<p>For the security of your personal data, Open Food Facts has implemented good security practices:</p>
+<ul>
+<li>Password hashing;</li>
+<li>Monitoring good practices in development;</li>
+<li>Authentication by SSH key, access to servers limited to a small trusted group, limitation of the attack surface, use of firewalls, etc.</li>
+</ul>
+<ol start="7">
+<h1>How to exercise your rights?&nbsp;</h1>
+</ol>
```
Comment (teolemon) [inline lang/en/texts/privacy.html:360]:
```suggestion
<h1>7. How to exercise your rights?&nbsp;</h1>
```

File: lang/en/texts/privacy.html [hunk 16]
```diff
@@ -0,0 +1,417 @@
...
+<p>To exercise one or more of <a href="#h.79sf63lc88my">your rights</a>&nbsp;, you can contact us by email: <a href="mailto:privacy@openfoodfacts.org">privacy@openfoodfacts.org</a>.&nbsp;In order to avoid identity theft, Open Food Facts will authenticate and control your email address using its internal technical processes.</p>
+<p>In certain circumstances, particularly in the event of a judicial and/or administrative investigation, Open Food Facts may transmit your personal data to authorized public authorities. We will endeavor to within&nbsp;the strict limits of the law, to inform you in the event of transmission of your personal data and will attempt to decline any abusive or unfounded request.</p>
+<ol start="8">
+<li>
+<h1>Dispute resolution</h1>
+</li>
+</ol>
```
Comment (teolemon) [inline lang/en/texts/privacy.html:367]:
```suggestion
<h1>8. Dispute resolution</h1>
```

File: lang/en/texts/privacy.html [hunk 17]
```diff
@@ -0,0 +1,417 @@
...
+<p>In the event of a dispute between you and Open Food Facts regarding the processing of your personal data, please contact us by email at the following address <a href="mailto:privacy@openfoodfacts.org">privacy@openfoodfacts.org</a>. We will respond as soon as possible.</p>
+<p>However, in the event of an unsatisfactory response or no response within a maximum of three months, you have the right to contact the CNIL via the following link <a href="https://www.cnil.fr/fr/saisir-la-cnil">https://www.cnil.fr/fr/saisir-la-cnil</a>&nbsp;or <a href="https://www.cnil.fr/fr/plaintes">https://www.cnil.fr/fr/plaintes</a>&nbsp;</p>
+<ol start="9">
+<li>
+<h1>Updates to this policy</h1>
+</li>
+</ol>
```
Comment (teolemon) [inline lang/en/texts/privacy.html:374]:
```suggestion
<h1>9. Updates to this policy</h1>
```

File: lang/en/texts/privacy.html [hunk 18]
```diff
@@ -0,0 +1,417 @@
...
+<p>To ensure a better protection, Open Food Facts reserves the right to modify and/or adapt this confidentiality policy following regulations and internal best practices. For each substantial modification, Open Food Facts will inform you at least thirty days before their application.</p>
+<p>As a collaborative project, Open Food Facts remains available to receive your suggestions regarding this privacy policy, which you can send by email to <a href="mailto:privacy@openfoodfacts.org">privacy@openfoodfacts.org</a></p>
+<ol start="10">
+<li>
+<h1>What are your rights ?</h1>
+</li>
+</ol>
```
Comment (teolemon) [inline lang/en/texts/privacy.html:381]:
```suggestion
<h1>10. What are your rights ?</h1>
```

File: lang/en/texts/privacy.html [hunk 19]
```diff
@@ -0,0 +1,417 @@
...
+<tbody>
+<tr>
+<td colspan="1" rowspan="1">
+<p>In summary:&nbsp;Open Food Facts is a collaborative project whose aim is to promote food transparency by providing an open product database. To make this database available and allow the community to contribute to it, Open Food Facts processes and stores for a limited period, your personal data such as user name, user identifier, passwords, email address and IP for which you have the right of access and information, right of opposition, right of rectification of data, and the right to limitation of processing. If you plan to exercise one of these rights, you can contact us by email: <a href="mailto:privacy@openfoodfacts.org">privacy@openfoodfacts.org</a></p>
+</td>
+</tr>
+</tbody>
+</table>
+<ol start="1">
+<li>
```
Comment (teolemon) [inline lang/en/texts/privacy.html:13]:
```suggestion
```

File: lang/en/texts/privacy.html (hunk 1 above)
Comment (teolemon) [inline lang/en/texts/privacy.html:16]:
```suggestion
```

File: lang/en/texts/privacy.html [hunk 20]
```diff
@@ -0,0 +1,379 @@
+<p>Privacy Policy</p>
```
Comment (teolemon) [inline lang/en/texts/privacy.html:1]:
```suggestion
<h1>The Open Food Facts Privacy Policy</h1>
```

**Pull request merged**